Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -2850,7 +2854,26 @@ as modifying the License.
 
     <!-- XBUT: submission result dialog close button -->
     <string name="submission_result_button_dialog_close">OK</string>
+    <!-- ####################################
+              Heading Fragments
+        ###################################### -->
+    <string name="heading_submission_contact">TAN Anfrage per Telefonanruf</string>
```

Comment:
this headings should live next to the screens where they are used, like all the other strings, e.g. this one should be in the `submission_contact` block. So I would suggest a name like `submission_contact_accessibility_title`.
 Please also add the translation comments.